src/main/java/com/google/cloud/tools/gradle/appengine/core/DownloadCloudSdkTaskConsoleListener.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.google.cloud.tools.gradle.appengine.core;
+
+import com.google.cloud.tools.managedcloudsdk.ConsoleListener;
+
+public class DownloadCloudSdkTaskConsoleListener implements ConsoleListener {
+  @Override
+  public void console(String rawString) {
+    System.out.print(rawString);
```

Comment:
Check out Gradle's logging thing: https://docs.gradle.org/current/userguide/logging.html

Reply:
Printing to stdout automatically logs at the QUIET level it seems, and I didn't see anything in the gradle api that would let me log without automatically adding a newline (newlines are included in the `rawstring` passed in, so it looks really bad using either `println` or `logger.lifecycle()`).

Reply:
Yeah I think system.out.print mimics a console best.

Reply:
We talked about QUIET maybe not being a good level for it since the `ConsoleListener` output is pretty noisy, so maybe we could buffer the raw string until we hit a newline and then use `logger.lifecycle()`.

I'm going to merge this and just submit another issue for that, though, if you think it's necessary.